plugins/kotlin-dataframe/src/org/jetbrains/kotlinx/dataframe/plugin/interpret.kt

```diff
@@ -269,6 +270,10 @@ fun <T> KotlinTypeFacade.interpret(
     }
 }
 
+fun interpretationFrameworkError(message: String): Nothing = throw InterpretationFrameworkError(message)
```

Comment:
the name looks like object name, not a function

Reply:
Can't argue, but i was inspired by "error" function from standard library